dpar-utils/src/stored_table.rs

```diff
@@ -76,6 +76,10 @@ impl Lookup for StoredLookupTable {
         }
     }
 
+    fn lookup_values(&self, values: &mut Vec<String>) {
+        //TODO: Implement
```

Comment:
Do not add unimplemeted functions. Or if it is absolutely necessary, use the `unimplemented` macro so that the code panics when a codepath reaching it is triggered:

https://doc.rust-lang.org/std/macro.unimplemented.html